src/App.jsx

```diff
@@ -1,13 +1,152 @@
-
 import './App.css'
+import {useState} from "react";
+import strawberry from "./assets/strawberry.png"
+import banana from "./assets/banana.png"
+import apple from "./assets/apple.png"
+import kiwi from "./assets/kiwi.png"
 
 function App() {
+    const [counter, setCounter] = useState({Aardbei: 0, Banaan: 0, Appel: 0, Kiwi: 0})
+
+   const handleClick = (fruit, action) => {
+
+        setCounter((prevState) => {
+                return {
+                    ...prevState,
+
+                    [fruit]: action === 'up'
+                        ? counter[fruit] + 1 : counter[fruit] - 1
```

Comment:
```suggestion
                    [fruit]: counter[fruit] + (action === 'up' ? 1 : -1)
```
Om het iets korter te schrijven 😉